ddataflow/ddataflow.py

```diff
@@ -400,24 +411,23 @@ def print_status(self):
         Print the status of the ddataflow
         """
         if self._offline_enabled:
-            print("DDataflow is now ENABLED in OFFLINE mode")
-            print(
+            logger.debug("DDataflow is now ENABLED in OFFLINE mode")
+            logger.debug(
                 "To disable it remove from your code or unset the enviroment variable 'unset ENABLE_DDATAFLOW ; unset ENABLE_OFFLINE_MODE'"
             )
         elif self._ddataflow_enabled:
-            print(
+            logger.debug(
+                """
+                DDataflow is now ENABLED in ONLINE mode. Filtered data will be used and it will write to temporary tables.
                 """
-DDataflow is now ENABLED in ONLINE mode. Filtered data will be used and it will write to temporary tables.
-"""
             )
         else:
-            print(
+            logger.debug(
                 f"""
-DDataflow is now DISABLED. So PRODUCTION data will be used and it will write to production tables.
-Use enable() function or export {self._ENABLE_DDATAFLOW_ENVVARIABLE}=True to enable it.
-If you are working offline use export ENABLE_OFFLINE_MODE=True instead.
-
-"""
+                DDataflow is now DISABLED.
```

Comment:
might still be a good idea to retain the " PRODUCTION data will be used" message

Reply:
You will always get `PRODUCTION data will be used` when ddataflow is not enabled in the beginning of the startup.